modules/nf-core/mapad/index/main.nf

```diff
@@ -0,0 +1,35 @@
+process MAPAD_INDEX {
+    tag "$meta.id"
+    label 'process_single'
```

Comment:
```suggestion
    label 'process_medium'
```

As it is actually multi-threaded [^1]

[^1] https://nf-co.re/docs/contributing/modules#resource-requirements point 1